server/app.ts

```diff
@@ -119,6 +119,12 @@ Created default user
 	Password: ${DEFAULT_PASSWORD}
 **Delete this user after you have used it to set up your account**
 	`);
+
+	// Add default list of tags 
+	let defaultTagsList = new TagsList({
+		tags: ["hackgt", "breakfast", "lunch", "dinner", "visited-microsoft"]
```

Comment:
Are we sure we want these to be the default tags instead of something like an empty array or just `hackgt`?